src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostExprTyping.scala

```diff
@@ -361,6 +361,8 @@ trait GhostExprTyping extends BaseTyping { this: TypeInfoImpl =>
       // Others
       case PReceive(_) => false
 
+      case PUnpackSlice(s) => isPure(s)(strong) // may change in the future
```

Comment:
Is it not just `go(s)`? Unpacking does not change the purity of the expression.

Reply:
`go(s)` is indeed better, even though they are equivalent